tools/azure-rest-api-specs-examples-automation/ci.yml

```diff
@@ -0,0 +1,32 @@
+trigger: none
```

Comment:
Should we have this trigger on changes in this directory or do you want another pipeline that will verify the tool itself?

Reply:
Yes, I should add a CI for this package as well (run the tests etc.).

Should we name that verification CI `ci.yml`, and rename this "automation of collect samples" pipeline to e.g. `ci-automation.yml`?

Reply:
Yes I'd name the verification ci.yml and the automation just "samples-automation.yml" and that name should also be included as part of the pipeline name in devops under the automation folder.

Reply:
Got it.

Add CI as `tools - azure-rest-api-specs-examples-automation - ci`, public, tools folder
Add automation as `automation - samples-automation`, internal, automation folder